ons_alpha/jinja2/templates/pages/topics/topic_page.html

```diff
@@ -1 +1,45 @@
 {% extends "templates/pages/index_page.html" %}
+
+{% from "components/panel/_macro.njk" import onsPanel %}
+
+{% block main %}
+    <div class="ons-grid ons-js-toc-container ons-u-ml-no ons-u-mt-s">
+        <div class="ons-grid__col ons-col-8@m ons-col-12@s ons-u-p-no@xxs@m">
+            <section id="summary">
+                <h2>{{ _("Summary")}}</h2>
+                <p>{{ page.summary}}</p>
+            </section>
+        </div>
+    </div>
+
+    {% block preSubpages %}
+        <div class="ons-grid ons-js-toc-container ons-u-ml-no ons-u-mt-s">
+            <div class="ons-grid__col ons-col-8@m ons-col-12@s ons-u-p-no@xxs@m">
+                <section id="topic_page_nav">
+                    {% for key in subpage_dict.keys() %}
+                        <a href="#{{ key }}">{{ key }}</a>
+                    {% endfor %}
```

Comment:
Minor, would be good to show the types that aren't our internal page names like "Article series page" vs "Articles" or "Articles related to {topic}" etc.

Reply:
**Hard coded** with default to humanreadable page type
https://docs.wagtail.org/en/stable/reference/pages/model_reference.html#wagtail.models.Page.page_type_display_name